src/Tribe/Commerce/Currency.php

```diff
@@ -838,19 +839,43 @@ public function get_formatted_currency_with_symbol( $amount, $post_id, $provider
 			}
 		}
 
-		$formatted = sprintf( $format, $currency['symbol'], $amount );
+		$formatted = sprintf( $format, $currency['symbol'], $amount ) . $suffix;
 
 		/**
 		 * Filter the Formatted Currency with Symbol
 		 *
 		 * @since 4.11.0
+		 * @since TBD Added $suffix param.
 		 *
 		 * @param string  $formatted The formatted amount.
 		 * @param int     $amount    The original amount to be formatted.
 		 * @param array   $currency  An array of currency formatting details.
 		 * @param boolean $html      Whether to return with html wrap.
+		 * @param string  $suffix    String to append to formatted amount.
 		 */
-		return apply_filters( 'tribe_format_amount_with_symbol', $formatted, $amount, $currency, $html );
+		return apply_filters( 'tribe_format_amount_with_symbol', $formatted, $amount, $currency, $html, $suffix );
 	}
 
-}
+	/**
+	 * Get the amount suffix, such as for WooCommerce's taxation suffix.
+	 *
+	 * @since TBD
+	 *
+	 * @param int|string  $amount
+	 * @param string|null $provider
+	 *
+	 * @return string
+	 */
+	private function get_amount_suffix_html( $amount, $provider = null ) {
+		$suffix = '';
+
+		if (
+			'Tribe__Tickets_Plus__Commerce__WooCommerce__Main' === $provider
+			&& method_exists( 'WC_Product', 'get_price_suffix' )
+		) {
+			$suffix = ( new WC_Product() )->get_price_suffix( $amount );
```

Comment:
Shorthand is great, readable is better

```suggestion
			$product = new WC_Product();
			
			$suffix = $product->get_price_suffix( $amount );
```